src/index.js

```diff
@@ -25,30 +25,45 @@ export default declare(({
     SVG_NAME,
     SVG_CODE,
     SVG_DEFAULT_PROPS_CODE,
+    EMIT_DEPRECATED_DEFAULT_PROPS,
   }) => {
+    const defaultProps = SVG_DEFAULT_PROPS_CODE
+      ? 'var props = objectAssign({}, SVG_DEFAULT_PROPS_CODE, overrides);'
+      : '';
+    const PROPS_NAME = SVG_DEFAULT_PROPS_CODE ? 'overrides' : 'props';
```

Comment:
where does "overrides" come from? is that a react thing?

Reply:
Nop, thats just renamed props, I didn't want to change the body (`SVG_CODE`) so I made a little rename here, the final function will be 

```js
var SVG_NAME = function SVG_NAME(overrides) {
  var props = objectAssign({}, SVG_DEFAULT_PROPS_CODE, overrides);
  
  return SVG_CODE // which will probably ref `props`, which I didn't want to mess with so overrides is just a rename
} 
```